autoscale-stack-launch-type.rst

```diff
@@ -0,0 +1,197 @@
+==========================
+ Auto-scaling Heat stacks
+==========================
+
+Brief summary
+=============
+
+In the *Event and schedule-based auto-scaling with Cloud
+Orchestration* tutorial, we used the "server" launchConfiguration type
+to scale the Cloud Servers in your web application.  Rackspace Auto
+Scale also supports a "stack" launchConfiguration type, where the unit
+of scale is a stack instead of a server.
+
+In this tutorial, we will learn how to scale a Heat stack, which will
```

Comment:
much clearer, though I would change this a bit to be:

  In this tutorial, we will learn how to scale using a group of Heat stacks which allows us to scale configurations more complex than a single server. In this example, we will use a stack containing both a server and a ``OS::Heat::SoftwareConfig`` to configure new instances in the group.